AnimatedComponents/src/components/buttons/icon-transition-button/IconTransitionButton.tsx

```diff
@@ -116,30 +116,31 @@ const IconTransitionButton = (props: IIconsTransitionButton) => {
     const maxTranslationX = buttonWidth.current - 50
     translateX.value = Math.min(maxTranslationX)
     const rotateDuration = 1500
-    const halfwayDuration = 750
+    const halfwayDuration = 950
```

Comment:
but its not halfway duration anymore then if its value is 950?? don't resolve comment just for sake of resolving be logical and take you time , code quality matters for your growth.